modules/test/playwright/tests/api-builder/apiFilter.spec.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {APIResponse, expect as baseExpect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const test = mergeTests(apiHelpersTest, loginTest);
+
+const expect = baseExpect.extend({
+	toBeSuccessful: (response: APIResponse) => ({
+		message: () =>
+			response.ok()
+				? 'Response is successful'
+				: 'Response is not successful',
+		pass: response.ok(),
+	}),
+	toHaveItems: async (response: APIResponse, expectedItems: object[]) => {
+		baseExpect((await response.json()).items).toEqual(expectedItems);
+
+		return {
+			message: () => 'The items are correct',
+			pass: true,
+		};
+	},
+});
+
+test('can GET with API Filter', async ({apiHelpers}) => {
+	await apiHelpers.featureFlag.updateFeatureFlag('LPS-178642', true);
```

Comment:
Take a look at the `headlessBuilderTest.ts` fixture. Here the feature flag is added and removed automatically before and after executing the tests. 

Also, see that in there there's also a piece of code that prevents the test from being executed until the API Builder Object definitions are created. This suggests me once more that the tests should be within the headless-builder Playwright test modules.

Reply:
since there's only one test in this file for now I din't want to overcomplicate it :D